sdk/spring-sdk/src/main/java/kalix/springsdk/annotations/Subscribe.java

```diff
@@ -27,6 +27,13 @@
     Class<? extends kalix.javasdk.valueentity.ValueEntity<?>> value();
   }
 
+  @Target({ElementType.TYPE, ElementType.METHOD})
```

Comment:
```suggestion
  @Target(ElementType.METHOD)
```
I think we should have only `ElementType.METHOD`. 

The reason why we have `TYPE` for ValueEntities is becaue we can have a View without transformation. When used without transformation, there is no method to transform the state from `Foo` to `ViewFoo` and then the annotation can be added at class level. 

For event sources, it will always be from one or many events to some other derived view state. So, by definition we need to have event handler methods for each event.

Reply:
understood 👍